install.sh

```diff
@@ -31,8 +31,8 @@ else
 	deno_uri="https://github.com/denoland/deno/releases/download/${1}/deno_${os}_x64.gz"
 fi
 
-deno_dir=${DENO_DIR:-$HOME/.deno}
-bin_dir="${deno_dir}/bin"
+deno_home=${DENO_HOME:-$HOME/.deno}
```

Comment:
Should this fall back to `DENO_DIR` if `DENO_HOME` is not specified?

Reply:
@j-f1 Well the point is draw a line between the two i.e. `DENO_HOME` is exclusively the installation directory, `DENO_DIR` is the cache.

For backwards compatibility? That would work for my use case, which is to set `DENO_HOME` to /opt/deno in my profile and generally let `DENO_DIR` default, but I think it would muddy the meanings of these variables. I'm not sure how many of whatever users there are at the moment are doing custom directory installations like this anyway.